File: src/main/java/com/temelyan/pomoapp/to/BaseTo.java
package com.temelyan.pomoapp.to;


import com.temelyan.pomoapp.HasId;

abstract public class BaseTo implements HasId {
    protected Integer id;

    BaseTo() {
    }

    BaseTo(Integer id) {
        this.id = id;
    }

    @Override
    public Integer getId() {
        return id;
    }

    @Override
    public void setId(Integer id) {
        this.id = id;
    }
}

File: src/main/java/com/temelyan/pomoapp/to/BaseTo.kt
package com.temelyan.pomoapp.to


import com.temelyan.pomoapp.HasId

abstract class BaseTo : HasId {
    private var id: Int? = null

    internal constructor()

    internal constructor(id: Int?) {
        this.id = id
    }

    override fun getId(): Int? {
        return id
    }

    override fun setId(id: Int) {
        this.id = id
    }
}
